The Stylex ESlint plugin README file does not seem to be up to date

### Describe the issue

The Stylex ESLint plugin README  says this is the way to set up the plugin:

```
{
  "rules": {
    "stylex/valid-styles": "error"
  },
  "plugins": ["stylex"]
}
```
But this will throw the following error:

```
Failed to load plugin 'stylex' declared in '.eslintrc.js': Cannot find module 'eslint-plugin-stylex'
```

### Expected behavior

There should be no error after following the README.

### Steps to reproduce

Follow the ESLint plugin README steps.

### Test case

_No response_

### Additional comments

The plugin documentation on the website is ok. However, I would change 

```
"@stylexjs/valid-styles": ["error", {...options}],
```

to 

```
"@stylexjs/valid-styles": "error"
```

As there is no `options` defined in the doc itself. Bad for a copy past. :)

--- a/apps/docs/docs/learn/03-installation.mdx
+++ b/apps/docs/docs/learn/03-installation.mdx
@@ -297,7 +297,7 @@ your styles.
 module.exports = {
   plugins: ["@stylexjs"],
   rules: {
-    "@stylexjs/valid-styles": ["error", {...options}],
+    "@stylexjs/valid-styles": "error",
   },
 };
 ```

--- a/packages/eslint-plugin/README.md
+++ b/packages/eslint-plugin/README.md
@@ -28,20 +28,20 @@ Once you've installed the npm package you can enable the plugin and rules by ope
 ```json
 {
   "rules": {
-    "stylex/valid-styles": "error",
-    "stylex/sort-keys": "warn"
+    "@stylexjs/valid-styles": "error",
+    "@stylexjs/sort-keys": "warn"
   },
-  "plugins": ["stylex"]
+  "plugins": ["@stylexjs"]
 }
 ```
 
 ## All the rules
 
-### stylex/valid-styles
+### @stylexjs/valid-styles
 
 StyleX requires styles that are statically analyzable. This rule will detect invalid styles that stylex cannot handle.
 
-### stylex/sort-keys
+### @stylexjs/sort-keys
 
 This rule helps to sort the StyleX property keys according to
 [property priorities](https://github.com/facebook/stylex/blob/main/packages/shared/src/utils/property-priorities.js).

